src/content/docs/en/reference/modules/astro-actions.mdx

```diff
@@ -173,4 +173,95 @@ The `code` property accepts human-readable versions of all HTTP status codes. Th
 <Since v="4.15.0" />
 </p>
 
-The `message` property accepts a string. (e.g. "User must be logged in.")
\ No newline at end of file
+The `message` property accepts a string. (e.g. "User must be logged in.")
+
+### `getActionContext()`
+
+<p>
+
+**Type:** `(context: APIContext) => ActionMiddlewareContext`
+<Since v="5.0.0" />
+</p>
+
+You may choose to customize how action results are handled. `getActionContext()` lets you programmatically get and set action results using middleware, allowing you to persist action results from HTML forms, gate action requests with added security checks, and more.
+
+Call `getActionContext()` from your middleware handler to retrieve information about inbound action requests. This function returns an `action` object with information about the request, and the `setActionResult()` and `serializeActionResult()` functions to programmically set the value returned by `Astro.getActionResult()`:
```

Comment:
Sorry @bholmesdev, it seems I can't push directly to the branch: permissions issue... or I don't know how to do it. So I'm not sure making a PR targeting this branch is easier just for a small change. 😅 After that, I think it will be good for me unless there are other changes and it needs another proofread! 😉 

typo:
```suggestion
Call `getActionContext()` from your middleware handler to retrieve information about inbound action requests. This function returns an `action` object with information about the request, and the `setActionResult()` and `serializeActionResult()` functions to programmatically set the value returned by `Astro.getActionResult()`:
```

Reply:
I got you!